### PR TITLE
Fix naming - get rid of underscore

### PR DESCRIPTION
**Pull request by pomozoff**
#trivial

### CODE REVIEW

File: Package.swift [hunk 1]
```diff
@@ -2,20 +2,20 @@
 import PackageDescription
 
 let package = Package(
-    name: "NSObject-Rx",
+    name: "NSObjectRx",
```
Comment (ashfurrow) [inline Package.swift:5]: The CocoaPod and Xcode project both use `NSObject+Rx`

https://github.com/RxSwiftCommunity/NSObject-Rx/blob/288e2c66567acf7ca0e42da95ab41f61f6a22c88/NSObject%2BRx.podspec#L2

https://github.com/RxSwiftCommunity/NSObject-Rx/blob/288e2c66567acf7ca0e42da95ab41f61f6a22c88/NSObject-Rx.xcodeproj/project.pbxproj#L11

Could we use that with SPM too? Or does it not like `+` in a package name?

Comment (pomozoff) [reply Package.swift:5]: Unfortunately, Swift converts minus (and probably plus as well) to underscore, so you have to write such mess: `import NSObject_Rx`
All I want is to change it to `import NSObjectRx` in the sake of consistency with language rules.

Comment (ashfurrow) [reply Package.swift:5]: That makes sense – the same thing happens with CocoaPods (and, as far as I'm aware, to Carthage as well). I know it's weird, and I think if I were starting the library today I'd use `NSObjectRx`, but as we already have `NSObject_Rx`, I think consistency would be more import than removing the underscore.

I'm open to feedback though. What do you think?

Comment (pomozoff) [reply Package.swift:5]: It is import to be consistent. If this import has the underscore with CocoaPods & Carthage, let it be with this underscore.

Comment (ashfurrow) [reply Package.swift:5]: Gotcha, that makes sense. Thanks! I'll close the PR.

**Pull request closed without merging**